lib/src/universal_platform_none.dart

```diff
@@ -0,0 +1,20 @@
+import 'universal_platform_type.dart';
+
+abstract class UniversalPlatform {
+  static UniversalPlatformType get value => UniversalPlatform.current;
+  static UniversalPlatformType get current => UniversalPlatformType.None;
```

Comment:
Are you sure this is correct? Why have both value and current? Why is current set to value, and none set to current?

Reply:
Hmmm, I was thinking of having .current as an alias to .value, but it might be unneeded. I'll remove it.